api/api-invoices/src/invoices/toSheetLine.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable prefer-destructuring */
+const { omit } = require('lodash')
+const format = require('date-fns/format')
+
+const toLine = (invoice, isTimetable, index) => {
+  const newInvoice = omit(invoice, ['ranges'])
+  const name = invoice.client.name
+  const id = invoice.id.includes('draft-') ? invoice.id : Number.parseInt(invoice.id, 10)
```

Comment:
why don't always convert to Int ?

Reply:
Because `draft-` is not convertable to int

Reply:
so, why convert to Int on the other case ?

Reply:
Because we do `id + 1` to retrieve next id in our spreadsheet.
Invoices ID should be incremental, and sequential, without "hole" in the sequence.